### DISCUSSION TITLE
Are there any examples of note JSON objects?

### DISCUSSION THREAD

**Discussion by manicphase** [Automattic/wordpress-activitypub#798] [Q&A] [answered]
I'm currently writing an activitypub compatible blog app for Django. I know most fediverse apps agree to use the content part of the note, but does wordpress have any special fields?

If I want to break a blog post out into fields such as 'title', 'body', 'synopsis' etc, I want it to be in alignment with whatever this plugin is doing.

**Comment by pfefferle** [comment, marked as answer]
Here you can find a:

* Note: https://notiz.blog/2018/02/02/a-letter-about-google-amp/?activitypub
* Article: https://notiz.blog/2024/04/26/activitypub-the-evolution-of-rss/?activitypub

**Reply by manicphase** [reply]
Thanks. It's not as complex as I expected after all
